src/components/inline/InputText/InputText.tsx

```diff
@@ -24,7 +23,20 @@ import s from "./InputText.module.css";
  *
  */
 
-const InputText = ({ name, placeholderText, onChange, value, type }) => {
+type InputTextProps = {
```

Comment:
For the purposes of defining React props, type aliases and interfaces are functionally interchangeable, so although I don't think we have a style guide yet, I think when we do create it, "we prefer to use type aliases over interfaces when possible" would be the kind of thing I'd want to see there. It doesn't make a difference to me, I'd just like us to pick one and stick with it.

Reply:
Yeah, I was wondering about that when I typed this line. Since I haven't had much real world experience here, I don't feel that I have a good sense of which to use. In other languages, I would normally prefer interfaces for anything that is actually used by another file, and aliases only for things where I want to save typing, but I think TypeScript's aliases are more powerful than type aliases in other languages.

I've found https://github.com/typescript-cheatsheets/react to be a pretty useful resource for best practices, and they have a specific section on [types vs. interfaces](https://github.com/typescript-cheatsheets/react/tree/cfa1209cdf926ab5894f0ef150c2fe48cce06a6b#types-or-interfaces). Looks like their rule of thumb is to use `type` for React props and state, but it sounds like they aren't dogmatic about it because there are causes where interfaces have advantages over types.

If that sounds reasonable to you, I think we can go with `type`s by default for props/state, but we can feel free to use `interface`s when it's useful.

Reply:
Yeah I'm happy using `type` by default -- I don't have strong opinions here, but I lean more towards `type` than I do against it.

I do think it can be confusing for people coming in from other languages, and older React/Typescript documentation and guides from before `type` was added still use `interface`. But as long as we write down somewhere that this was a style choice we made, I don't foresee any problems.

Reply:
Sounds good! Let me leave this PR open for another day, in case if anyone one else wanted to do a review on it, but if there are no other comments, I can merge it in tomorrow.